TYPO3.Neos/Documentation/ExtendingNeos/InteractionWithTheNeosBackend.rst

```diff
@@ -37,6 +37,8 @@ events occur:
 * **Neos.PageLoaded** Whenever the page reloads by Ajax.
 * **Neos.PreviewModeActivated** When the backend switches from edit to preview mode.
 * **Neos.PreviewModeDeactivated** When the backend switches from preview to edit mode.
+* **Neos.FullScreenModeActivated** When the backend switches to fullscreen mode.
+* **Neos.FullScreenModeDeactivated** When the backend switches from fullscreen to none fullscreen mode.
```

Comment:
I argue that this qualify as a LayoutChanged event as well? So it should be grouped with those events and not next to PreviewMode

Additionally it should trigger the LayoutChanged event of course